plugin/PDFSelectionHandle.qml

```diff
@@ -16,7 +16,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-import QtQuick 2.0
+import QtQuick 2.6
```

Comment:
Not related to this line, but bug: have another handle near the left side of the display, try to grab and move it horizontally -> the whole page gets dragged.

Reply:
Fixed in Silica side. DragDetectorItem didn't respect preventStealing.